test/custom-worker.js

```diff
@@ -0,0 +1,60 @@
+// This example uses @typescript/vfs to create a virtual TS program
+// which can do work on a bg thread.
+
+// This version of the vfs edits the global scope (in the case of a webworker, this is 'self')
+importScripts("https://unpkg.com/@typescript/vfs@1.3.0/dist/vfs.globals.js")
+
+/** @type { import("@typescript/vfs") } */
+const tsvfs = globalThis.tsvfs
+
+/** @type {import("../src/tsWorker").CustomTSWebWorkerFactory }*/
+const worker = (TypeScriptWorker, ts, libFileMap) => {
```

Comment:
This should be everything people need to either re-create the TypeScript worker themselves from scratch or to augment it with new functions I think